setup.py

```diff
@@ -4,7 +4,9 @@
 from setuptools import Extension
 import os
 
-VERSION="1.1"
+VERSION='undefined'
+with open('sourmash_lib/version.py') as f:
+    exec(f.read())
```

Comment:
Hmmmm, can we use the other alternative from that answer, the one that doesn't need `exec()`?